client/components/mediaCarousel/mediaCarousel.js

```diff
@@ -128,9 +131,9 @@ class MediaCarousel extends React.Component {
         }
 
         <ImagesDiv>
-          {this.state.currentArr.map(media => {
+          {this.state.currentArr.slice(0, 6).map(media => {
             return (
-            <Image src={media.thumbnail || media.link} key={media.id} id={media.id} alt='dummy data' width={96} height={53.75} onClick={(e) => imageClickHandler(e)} />
+            <Image key={media.id} id={media.id} alt='dummy data' onClick={(e) => imageClickHandler(e)}  src={media.thumbnail || media.link} width={96} height={53.75} />
```

Comment:
This just looks like a rearrangement of these props. Was there a reason for that? Or just code organization?

Reply:
yes, what you're seeing was an attempt to add something in but I ended up just going back to this